loc/US/strings_db.lua

```
@@ -5639,7 +5639,7 @@ tooltipui0398="Click here to send your message to all of your allies."
 tooltipui0399="All Chat"
 tooltipui0400="Click here to send your message to all players."
 tooltipui0401="Chat Filters"
-tooltipui0402="Show or hide messages and paintings from players"
+tooltipui0402="Show or hide messages from players"
```

The default is already `Show or hide messages from players`